tests/fixtures/datacite/zenodo_records_containing_unaccepted_content_types.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<records>
```

Comment:
I would remove all the optional fields except `resourceType` from this record since they aren't used by the test and make it harder to understand the difference between the two records.